### ISSUE TITLE
Revise draft homepage principles based on FEC feedback

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-cms#517]
After presenting the [first draft of the principle list](https://docs.google.com/a/gsa.gov/document/d/1bzKtjKZPXnpCxCW00S6EVgQiW577EIW5Yi9NLhRu8Do/edit?usp=sharing) on 9/22, and receiving feedback collected by Amy & the FEC team, we should revise the principles and prepare a new draft to bring to our in-person workshops.

**Comment by jenniferthibault** [comment]
Note from sprint planning: use visual tools such as concept sketches (or other) to facilitate conversation.

**Comment by jenniferthibault** [comment]
The main areas of feedback that came in are: 

**"Serve the largest groups best"**
We talked about this one in the first meeting last week, and it felt least on the mark.
There are user groups, like filers, who have a statutorily required relationship with the FEC. Since they are legally required to interact with the FEC, the FEC should make them a high priority group,   irrespective of the size of that group. 
> "Folks are uncomfortable with treating the size of the users base as the defining metric at the expense of considering other users' level of involvement (and risk) in the campaign finance process. We believe we owe it to filers to serve them at least equally as well as members of the general public."  

**"Help users self-identify which tasks apply to them"**
This one also came up at the first meeting with a little follow-up after, because it seems like we are closing the door too much on helping users self-identify what FEC user group they are. In order to perform all of the requirements for FEC activities, users must eventually learn what group they fall into in the world of campaign finance activity. We could investigate different orders to facilitate this learning. 

For example, I've been thinking about it in terms of which order you put the information in. We can use a sentence to demonstrate, but that can also happen visually, with how you present information:

"`I am a ________.` `That means I need to _______`"
or 
"`I need to _________.` `That's grouped with other tasks in the ________ category, so I must be a _______`"
or 
"`I am looking for a __________` `so that I can __________.` `That must mean I'm a _________`" 

The three units are tasks, audience type, and object type. We can try different orders and test them with users to find which they navigate the easiest.

@emileighoutlaw is on deck to wordsmith the principles into shape that will help keep the discussion open when we are in person with the FEC. We'll still consider them a working draft through that point.


To start prepping visual tools for the principles conversation, I started [this Mural](https://app.mural.ly/t/gsa6/m/gsa6/1475187277477) where I've broken down modules of things we can start sketching in different ways. 
- @nickykrause you may think of things I haven't, feel free to add!

Today I'm going to start collecting examples of modules on other sites that do this. I'll drop them into the mural.

**Comment by nickykrause** [comment]
@jenniferthibault Thank you for the excellent feedback summary!

I love this: 

> "I am a ________. That means I need to _______"
or 
"I need to _________ so I must be a _______"
or 
"I am looking for a __________ so that I can __________. That must mean I'm a _________"
The three units are tasks, audience type, and object type. We can try different orders and test them with users to find which they navigate the easiest.

I think it is a very helpful way of communicating the problem and possible approaches to solving it. 

I will check out the Mural!

**Comment by emileighoutlaw** [comment]
Hello there! I'm ready for some feedback!

When I pulled apart and reassembled all the principles, everything under “serve the largest group best” fit neatly into the other six principles. Since there’s no rule that we have to start and end with seven principles, I went with this reshuffling (and a few others) to see how things shook out. I actually think this version might get at all the things we care about. [[Revised] Principles](https://docs.google.com/document/d/1bzKtjKZPXnpCxCW00S6EVgQiW577EIW5Yi9NLhRu8Do/edit#) are on the first page of the `[Draft] Principles` document.

I don't think we're 100% there yet, but I think we're getting close. Feel free to mosey over to that doc and take a look. Or, if you'd like, read a full accounting of my changes in the section that follows.

cc @nickykrause @noahmanger @jenniferthibault 

### Here’s what I did:

**Moved the subpoints of `Serve the largest group` into other sections**
- Moved _Users that generate data (committees) are not necessarily the same users that consume content_ into `Help users identify which tasks apply to them`
- Moved _Small groups of users may be particularly vocal and appear “needier,"..._ into `Prioritize`.
- Moved _Users have different levels of FEC fluency._ and _You can address different groups in different way and at different times._ to `Meet users at their level.`


**Reordered the principles**
Once our number 1 principle was gone, it felt like the list order needed reconsidering. I tweaked thing based on what my understanding of our priorities (and the challenges we want to face head on).


**Tweaked some of our phrasings and other subpoints**
All of these are definitely up for a gut check.
- _This will help people find their way around without needing to devise their own strategy for doing so._ felt a little opaque to me. So I made it a more concise _Don’t force people to devise their own navigation strategies_. The more concise version felt like it should live in `help users identify which tasks apply to them` so I moved it.
- _Users don't fit neatly into buckets. Many of them share overlapping goals. The homepage needs to acknowledge that people in different groups often need to accomplish the same tasks, without removing it from another group’s grasp._ also felt wonky to me. I changed it to _Users share overlapping goals. They don't fit neatly into buckets. Ensure that people in different groups can accomplish the same tasks, if needed._

**Comment by jenniferthibault** [comment]
Thank you so much for the lovely writeup, @emileighoutlaw ! Reading through made me feel like I made these updates right along with you.

I am totally in favor of fewer principles as a starting point, and love how you've pulled things together. Reading through made me realize that the remaining tension between `Provide a clear and consistent conceptual structure` and `Help users identify which tasks apply to them` could be resolved if we admit that identifying tasks is the organizing conceptual structure.

Since task-based nav vs audience-based nav seems to be a tension point, would it be too vague to instead write `Help users identify which tasks apply to them` as `Help users identify their goals` ?
That could mean that a goal is a task, or an exploration, but maybe I am pushing it to be too vague.

**Comment by nickykrause** [comment]
Responding to this, from @jenniferthibault: 

> Since task-based nav vs audience-based nav seems to be a tension point, would it be too vague to instead write Help users identify which tasks apply to them as Help users identify their goals ?

I think the underlying point here about taking out the word "tasks" does make sense, and it might not make the principle too vague to change the wording. 

On the idea of goals: We do have research to suggest that the users' "goals" most likely take the form of task completion. So, when we design against the principle of helping users accomplish their goals, we would lean on the research to understand what those goals are. (Meaning: The principle itself does not have to prescribe the goals as being task-oriented). 

But, I just said "helping users accomplish their goals," which is different than the wording of "help users identify their goals." I don't know if "goals" is what we are helping users to identify. We are maybe helping them to _identify information_ that will help them accomplish their goals, or to _identify pathways_ to accomplishing their goals...? Hm. I'm not sure. Also, in the latter case, it wouldn't just be about helping them identify _all_ the pathways, but the most _effective_ ones. 

Maybe something like "Help users to identify what content will meet their needs?"

**Comment by emileighoutlaw** [comment]
Awesome thoughts, y'all! Looking at at the sub-points under `Help users...`

> -Create easily recognizable paths to frequently referenced material. 
> -Users share overlapping goals. They don't fit neatly into buckets. Ensure that people in different groups can accomplish the same tasks, if needed.

sheds light on what principle I think we're actually trying to support. In this case, I think it's `Help users accomplish their goals.`

An important piece of accomplishing goals is identifying what content applies to them. There might be another bullet to be added to that point, if y'all agree.

**Comment by nickykrause** [comment]
@emileighoutlaw Great! yes, agreed

**Comment by jenniferthibault** [comment]
Agreed as well! I gave a last read-through & comments in the document.

I wanted to also give an update on the visual side of things: how we could use the mural of modules to talk about the principles. 

Here's an idea for an in-person exercise/discussion prompt:

- Form small groups with 18F designer:2-3 FEC team members
- Reintroduce revised principles: talk about how during this meeting we should talk about when one is helpful or confusing, then work on it there. "Living draft"
- Show murally: "We used your examples of sites to group things into "module" categories. I.e., things that do a type of thing. Then we found many different ways of doing that thing. For example, "A way to show what's going on at the agency" - you can do that through a calendar, a list of events, a featured event, a list of news items. Lots of different ways to do the same thing.

  - ![screen shot 2016-10-05 at 1 56 54 pm](https://cloud.githubusercontent.com/assets/11636908/19125021/7a660ee0-8b03-11e6-9999-5ef871e7c478.png)

- We want to start figuring out first what order these go in, then which way of doing it feels best.
- Start big: "Put these cards in order of how you think someone should experience them" What's the general flow of the page?
  - ![fullsizerender](https://cloud.githubusercontent.com/assets/11636908/19125102/cbbef96e-8b03-11e6-919b-1ebfe8f556a0.jpg)

We might get something like this:
  - ![fullsizerender 4](https://cloud.githubusercontent.com/assets/11636908/19125080/b6e11cac-8b03-11e6-8a21-f9c7b5c4d2ea.jpg)

 - "Then, consider what we said before about how you can do one thing many ways. Here are some traced patterns. Think of them like lego blocks. Which blocks would you use to make that thing, and how would you do it? Select blocks that get you a general arrangement that you like. Use pens to write inside each empty square with content that shows what would be on the homepage today"
![fullsizerender 3](https://cloud.githubusercontent.com/assets/11636908/19125128/e377552e-8b03-11e6-9731-29a5df56f38a.jpg)

- We'd end up with something that looks like this, but with copy/images sketched out. I didn't want to write all over the templates we'd use! Each group should present and talk about the arrangement they landed on, what content is in each module, and why they chose that way to do it.

  - ![fullsizerender 2](https://cloud.githubusercontent.com/assets/11636908/19125232/4955130e-8b04-11e6-9d1e-76f6023efed9.jpg)



From there, I'm of two minds of how we go with it. We could think of the session goals as
- creating discussion about order, priority, and principles
or
 - selecting one or two concept sketches to wireframe together

One focuses on process, participation, and open discussion, the other focuses on leaving with a product.

Would love all thoughts and feedback on this exercise, as well as what the goal of it should be. If it's ok with you all, we can also introduce this to the FEC team today when we talk through the revised draft principles, and see if they think this is a good format or activity to do in the first place.

**Comment by nickykrause** [comment]
@jenniferthibault and I talked about this earlier, and I just want to say, Jen, that you have already brought this so far in such a short time! The pictures are so helpful in understanding what the exercise would be and how it would work. Your ability to communicate visually is endlessly impressive to me. 

To your question at the end: I think I am leaning toward the emphasis on participation and discussion, rather than walking away with a product. There is a part of me that wonders if there will be a lot of pressure on the exercise itself to produce the perfect solution, if there is a sense afterward that the ideas will become our wireframe options. (But, perhaps that is not the idea?)

Also, I worry a bit about not having enough time to reflect, in the session, on whether or not the proposed/sketched ideas will indeed be workable. I don't want the partners to feel, coming out of the session, like we've settled on a wireframe option, and then we have to come back later and say it won't work for some reason? (But, maybe that's unlikely, and maybe that wouldn't be there expectation). 

I am also in favor of vetting the idea with the FEC folks. Maybe we frame it to them, though, as a recommendation (since it does indeed seem useful, and like something we ought to do) rather than just passing idea? Maybe then, if they have concerns about it, we have more hope of refining it into something we all can agree would be useful, rather than tossing out the idea of this kind of collaborative session?
